src/Test/Utilities/Portable/Diagnostics/DiagnosticDescription.cs

```diff
@@ -219,13 +217,8 @@ public override bool Equals(object obj)
                 {
                     if (_startPosition.Value != d._startPosition.Value)
                     {
-                        _showPosition = true;
-                        d._showPosition = true;
```

Comment:
@DavidKarlas i could forgive it in GetHashCode and ToString *as long as*  it is just a aching a lazily computed value to prevent recalc